conformance/system_macros/add_macros.ion

```diff
@@ -1,9 +1,87 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Test Cases:
-// add_macros can be invoked using any type of macro reference.
-// the argument values may be zero or more macro definitions
-// add_macros does not have any side-effects on the symbol table
-// add_macros should append its arguments to the macro table of the default module
-// annotations on any argument value should signal an error (because it's not allowed in the macro table rather than because of any specific validation by the add_macros macro)
+// TODO: add_macros can accept a module name (of a loaded module) as an argument?
```

Comment:
Yes, I think so. The options would be:
* **`(macro name () body)`** to define a macro in-situ.
* **`macro_ref`** (qualified or unqualified) to add an already-defined macro
* **`module_name::*`** to add all of `module_name`'s exported macros

Note that `foo` would be considered an unqualified macro name rather than a module, which is why `foo::*` exists.